packages/cli/src/services/public-api-key.service.ts

```diff
@@ -28,8 +29,11 @@ export class PublicApiKeyService {
 	 * Creates a new public API key for the specified user.
 	 * @param user - The user for whom the API key is being created.
 	 */
-	async createPublicApiKeyForUser(user: User, { label }: { label: string }) {
-		const apiKey = this.generateApiKey(user);
+	async createPublicApiKeyForUser(
```

Comment:
Just noticed, the name of this function is a bit misleading, as we do upsert here instead of just insert. Maybe it could be `createOrUpdate...` or `upsert...`?

Reply:
If I remember correctly, I did `upser`t because it returned the record back, unlike the `save` operation.